### PR TITLE
Updated attributes

### PR DESCRIPTION
**Pull request by TamimiGitHub**

### CODE REVIEW

File: docs/SupportedCommands/Commands.md [hunk 1]
```diff
@@ -79,6 +79,8 @@ Examples (Ruby):
 | `id`                       | 					string			         | 					4.5.1+	 				|
 | `isDisplayed`              | 					bool	 				       | 					4.5.1+	 				|
 | `isHittable`               | 					bool	 				       | 					4.5.1+	 				|
+| `isHorizontalScrolling`               | 					bool	 				       | 					4.5.1+	 				|
```
Comment (simongranger) [inline docs/SupportedCommands/Commands.md]: This should wait until it goes in.

Comment (TamimiGitHub) [reply docs/SupportedCommands/Commands.md]: Sure 👍

**Comment by TamimiGitHub** [comment]
Updated

File: docs/SupportedCommands/Commands.md [hunk 2]
```diff
@@ -78,7 +78,10 @@ Examples (Ruby):
 | `hasFocus`                 | 					bool	 				       | 					4.5.1+	 				|
 | `id`                       | 					string			         | 					4.5.1+	 				|
 | `isDisplayed`              | 					bool	 				       | 					4.5.1+	 				|
+| `isFullyDisplayed`               | 					bool	 				       | 					4.6.0+	 				|
```
Comment (simongranger) [inline docs/SupportedCommands/Commands.md]: won't this go in 4.5.2?

File: docs/SupportedCommands/Commands.md [hunk 3]
```diff
@@ -78,7 +78,10 @@ Examples (Ruby):
...
 | `isHittable`               | 					bool	 				       | 					4.5.1+	 				|
+| `isHorizontalScrolling`               | 					bool	 				       | 					4.6.0+	 				|
```
Comment (simongranger) [inline docs/SupportedCommands/Commands.md]: 4.5.2?

File: docs/SupportedCommands/Commands.md [hunk 4]
```diff
@@ -78,7 +78,10 @@ Examples (Ruby):
...
+| `isScrolling`               | 					bool	 				       | 					4.5.1+	 				|
```
Comment (simongranger) [inline docs/SupportedCommands/Commands.md]: was it already there?

File: docs/SupportedCommands/Commands.md (hunk 3 above)
Comment (TamimiGitHub) [reply docs/SupportedCommands/Commands.md]: No, it will be in 4.6+ because that what develop is forked to right now... Could be cherry-picked though

Comment (simongranger) [reply docs/SupportedCommands/Commands.md]: I would cherry-pick to 4.5.2

**Pull request merged**